src/Factories/FunFunFactory.php

```diff
@@ -468,6 +471,10 @@ function( string $name, array $parameters = [] ): string {
 			return new SofortClient( $config['config-key'] );
 		};
 
+		$pimple['skin'] = function () {
```

Comment:
Since `skin` is [more of a parameter than a service](https://pimple.symfony.com/#defining-parameters) I think it's sufficient to assign it directly instead of using a function with a return value.